Terasology/Economy
# Add currency to the module

**To Test**
 - Start up a game with Economy enabled
 - Should see a new HUD widget on the left, showing the player's money
 - Money in the wallet can be changed by firing the UpdateWalletEvent

![wallet](https://user-images.githubusercontent.com/32035777/61874888-17c61480-af07-11e9-97b2-ec5508799425.png)
